tornado/test/template_test.py

```diff
@@ -208,10 +208,15 @@ def test_error_line_number_directive(self):
             self.assertTrue("# test.html:2" in traceback.format_exc())
 
     def test_error_line_number_module(self):
+        loader = None
+
+        def load_generate(path, **kwargs):
+            return loader.load(path).generate(**kwargs)
+
         loader = DictLoader({
             "base.html": "{% module Template('sub.html') %}",
             "sub.html": "{{1/0}}",
-        }, namespace={"_tt_modules": ObjectDict(Template=lambda path, **kwargs: loader.load(path).generate(**kwargs))})
+        }, namespace={"_tt_modules": ObjectDict(Template=load_generate)})
```

Comment:
this is the biggest structural change I did, for "fixing" long lines in this case ... if you don't prefer this I can just "# noqa" it